styles.css

```diff
@@ -0,0 +1,54 @@
+
+body,html{
+    background-color: white;
+
+
+
+}
+main{
+
+    display: flex;
+    flex-wrap: wrap;
+    background-color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content:  center;
+
+
+}
+div {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+align-items: center;
+align-content: center;
+}
+
+#image {
+    max-width: 100%;
+    max-height: 100%;
+    
+    display: block;
+}
+
+
+
+
+ul,h2{
+    width: 50%;
+}
+a{
+    display: flex;
+    width: 100%;
+    justify-content: center;
+}figcaption{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+h1{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
```

Comment:
Kindly follow the HTML best practices regarding indentation and black lines for better readability of your code.
Please consider:
Be careful with blank lines and indentation.
Do not add blank lines, spaces, or indentations without a reason.
For readability, add blank lines to separate large or logical code blocks.
For readability, add two spaces of indentation. Do not use the tab key.
on some code editors, you can set the tab button to do an X amount of spaces.